HEC.FDA.Model/alternatives/Alternative.cs

```diff
@@ -204,19 +204,24 @@ private static AggregatedConsequencesByQuantile IterateOnAAEQ(
             int probabilitySteps = 25000;
             var resultCollection = new ConcurrentBag<double>();
 
+            //we're taking the EAD from base year and the most likely future year, and computing the EqAD for 25000 samples from the histograms, walking in small probabilty steps through both. 
```

Comment:
I wouldn't say that we take 25000 samples. 

I might write this as "We compute the nth quantile of EqAD from the nth quantile of the base year EAD distribution and the nth quantile of the most likely future year EAD distribution, where n is determined by walking through 25,000 probability steps."

I don't think this semantics, I think there is an important technical distinction here. We're not sampling the EAD distributions; we don't use Monte Carlo to compute EqAD.